pkg/option/config.go

```diff
@@ -2504,6 +2511,12 @@ func (c *DaemonConfig) Populate() {
 
 	c.populateDevices()
 
+	c.IPAMClusterPoolIPv4RoutingCIDR = viper.GetStringSlice(operatorOption.IPAMOperatorV4CIDR)
+	if len(c.IPAMClusterPoolIPv4RoutingCIDR) > 0 {
+		c.ipv4NativeRoutingCIDR = cidr.MustParseCIDR(c.IPAMClusterPoolIPv4RoutingCIDR[0])
```

Comment:
1. Please log an info msg that native routing cidr is derived from the ipma cluster ipv4 pool.
2. What's the reasoning for using the first element from the slice? Perhaps you want to add a code comment explaining the reasoning.

Reply:
ad 2. Actually I don't see the reason of `[]string{}` as a type of `IPAMClusterPoolIPv4RoutingCIDR`. I will change it just to `string`.